docs/GOVERNANCE.md

```diff
@@ -0,0 +1,139 @@
+
+among other possibilities. Any community member can become a contributor, and
+all are encouraged to do so. By contributing to the project, community members
+can directly help to shape its future.
+
+Contributors are encouraged to read the [contributing guide](CONTRIBUTING.md).
+
+## Core developers
+
+Core developers are community members that have demonstrated continued
+commitment to the project through ongoing contributions. They
+have shown they can be trusted to maintain napari with care. Becoming a
+core developer allows contributors to merge approved pull requests, cast votes
+for and against merging a pull-request, and be involved in deciding major
+changes to the API, and thereby more easily carry on with their project related
+activities. Core developers appear as organization members on the napari
+[GitHub organization](https://github.com/orgs/napari/people) and are on our
+[@core-devs](https://github.com/orgs/napari/teams/core-devs) GitHub team. Core
+developers are expected to review code contributions while adhering to the
+[core developer guide](CORE_DEV_GUIDE.md). New core developers can be nominated
+by any existing core developer, and for details on that process see our core
+developer guide.
+
+## Steering Committee
+
+The Steering Committee (SC) members are core developers who have additional
+responsibilities to ensure the smooth running of the project. SC members are
+expected to participate in strategic planning, approve changes to the
+governance model, and make decisions about funding granted to the project
+itself. (Funding to community members is theirs to pursue and manage.) The
+purpose of the SC is to ensure smooth progress from the big-picture
+perspective. Changes that impact the full project require analysis informed by
+long experience with both the project and the larger ecosystem. When the core
+developer community (including the SC members) fails to reach such a consensus
+in a reasonable timeframe, the SC is the entity that resolves the issue.
+
+The steering committee is currently fixed in size to N members. This number may be increased in
+the future, but will always be an odd number to ensure a simple majority vote outcome
+is always possible. The initial steering committee of napari consists of
+
+- XXXXXXXXX.
+...
+- XXXXXXXXX.
+
+The SC membership is revisited every January. SC members who do
+not actively engage with the SC duties are expected to resign. New members are
+added by nomination by a core developer. Nominees should have demonstrated
+long-term, continued commitment to the project and its [mission and values](MISSION_AND_VALUES.md). A
+nomination will result in discussion that cannot take more than a month and
+then admission to the SC by consensus.
+
+The napari steering committee may be contacted at `napari-steering-committee@googlegroups.com`.
+Or our via our [@steering-committee](https://github.com/orgs/napari/teams/core-devs) GitHub team.
```

Comment:
this still points to core-devs. is this intentional?

Reply:
no, need to create a "team"

Reply:
fixed